src/careamics/utils/metrics.py

```diff
@@ -186,3 +204,173 @@ def get(self) -> Optional[torch.Tensor]:
             return None
         rmse = torch.sqrt(self.mse_sum / self.N)
         return 20 * torch.log10((self.max - self.min) / rmse)
+
+
+def _range_invariant_multiscale_ssim(
```

Comment:
I think this needs to change. this is CARE-SSIM. (I know this existed in the Disentangle repository and so you copied it..).  However, MicroSSIM, the SSIM variant I report the numbers of is https://github.com/juglab/MicroSSIM

```
from microssim import MicroSSIM, MicroMS3IM
gt: N x H x W
pred: N x H x W

ssim = MicroSSIM() # or MicroMS3IM()
ssim.fit(gt, pred)

for i in range(N):
    score = ssim.score(gt[i], pred[i])
    print('SSIM score for', i, 'th image:', score)

```
Specifically, I use the following function to compute the numbers https://github.com/ashesh-0/Disentangle/blob/master/disentangle/scripts/evaluate.py#L127

Reply:
Can we hold off on that until we make MicroSSIM a dependency of CAREamics? Should happen next week.

Reply:
Sure, not an issue as the main metric to evaluate performance is PSNR!